types/three/src/textures/Texture.d.ts

```diff
@@ -22,7 +22,7 @@ export class Texture extends EventDispatcher {
      * @param [format=THREE.RGBAFormat]
      * @param [type=THREE.UnsignedByteType]
      * @param [anisotropy=1]
-     * @param [encoding=THREE.LinearEncoding]
+     * @param [encoding]
```

Comment:
From what I can tell, `encoding` still has a default from the constructor.

Reply:
yeah lets get rid of this PR then